docs/docker-swarm-usage.md

```diff
@@ -2,42 +2,46 @@
 
 ## Healthcheck
 
-The [Docker Image](../Dockerfile) includes a Healthcheck with the following options:
+The [Docker Image](../Dockerfile) includes a health check with the following options:
 
 ```
 --interval=30s
 ```
-> Specifies the time interval at which the health check should be performed. In this case, the health check will be performed every 30 seconds.
-
+> Specifies the time interval to run the health check. \
+> In this case, the health check is performed every 30 seconds.
 <br>
 
 ```
 --timeout=10s
 ```
-> Specifies the amount of time to wait for a response from the health check command. If the response does not arrive within 10 seconds, the health check will be considered a failure.
-
+> Specifies the amount of time to wait for a response from the \"healthcheck\" command. \
```

Comment:
Could be this is "health-check" or something else.(?)

Reply:
It actually seems to be uppercase `HEALTHCHECK` ([1](https://docs.docker.com/engine/reference/builder/#healthcheck))

Reply:
@comradekingu I would love to merge this. Could you first resolve this please! :)

Reply:
Seems good now.